tests/SpecTests/PrimaryStepDownSpecTest.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+namespace MongoDB\Tests\SpecTests;
+
+use IteratorIterator;
+use MongoDB\Client;
+use MongoDB\Collection;
+use MongoDB\Driver\Command;
+use MongoDB\Driver\Exception\BulkWriteException;
+use MongoDB\Driver\Exception\Exception as DriverException;
+use MongoDB\Driver\ReadPreference;
+use MongoDB\Driver\WriteConcern;
+use MongoDB\Operation\BulkWrite;
+use MongoDB\Tests\CommandObserver;
+use UnexpectedValueException;
+
+class PrimaryStepDownSpecTest extends FunctionalTestCase
```

Comment:
I might suggest a `@see` link to https://github.com/mongodb/specifications/tree/master/source/connections-survive-step-down/tests in a class-level doc block.